.gitpod.yml

```
@@ -1,23 +1,19 @@
 ---
-image: gitpod/workspace-base
+image:
+    file: .gitpod.Dockerfile
+
 # Commands that will run on workspace start 
 tasks: 
-    - name: Setup, Install & Build 
-      before: apt install cargo redis-server nodejs npm -y && cargo test
-      init: cargo install cargo-watch 
-      command: redis-server --port 8080 & cargo watch -q -w "." -x "run" 
-# Ports to expose on workspace startup 
-ports: 
-    - name: Website 
-      description: Website Preview 
-      port: 8080 
-      onOpen: open-preview
+    - name: Build & Deploy
+      init: cargo build -r
+      command: redis-server --port 8082 & ./target/release/websurfx
+
```

```suggestion
    - name: Start Redis Server
      command: redis-server --port 8082
    - name: Run The App
      init: cargo build
      command: PKG_ENV=dev ./target/release/websurfx
    - name: Tests
      command: cargo tests
    - name: Clippy Checks
      command: cargo clippy
```
Thanks :heart: for working on this, but I think having things run in separate terminals would be better. What do you say ?